addons/website/models/ir_http.py

```diff
@@ -158,6 +159,19 @@ def _register_website_track(cls, response):
 
         return False
 
+    @classmethod
+    def _postprocess_args(cls, arguments, rule):
+        super()._postprocess_args(arguments, rule)
```

Comment:
it is wrong, _postprocess_args from http_routing can return Response that you ignore.
Nice to see that my test from v14 slug_maching handle it ;)

Reply:
Indeed good catch.
Change done, I tested with your test and tests are OK with response return